doc/sphinx/proof-engine/ltac.rst

```diff
@@ -707,13 +707,25 @@ First tactic to succeed
 In some cases backtracking may be too expensive.
 
 .. tacn:: first [ {*| @ltac_expr } ]
+          first @ltac_expr
+   :name: first; _
+
+   In the first form: for each focused goal, independently apply the first
+   :token:`ltac_expr` that succeeds.
+
+   In the second form: :n:`@ltac_expr` is an :n:`@ident` that represents a list
+   of tactics passed to :n:`first` in a :cmd:`Tactic Notation` command (see example
+   :ref:`here <taclist_in_first>`).
+
+   :tacn:`first` is an :token:`l1_tactic`.
 
-   For each focused goal, independently apply the first :token:`ltac_expr` that succeeds.
-   The :n:`@ltac_expr`\s must evaluate to tactic values.
    Failures in tactics after the :tacn:`first` won't cause backtracking.
    (To allow backtracking, use the :tacn:`+<+ (backtracking branching)>`
    construct above instead.)
```

Comment:
This sentence looks like it should be moved closer to the paragraph below (between the `exn` and the example).